Packs/VMRay/ReleaseNotes/1_1_11.md

```diff
@@ -0,0 +1,21 @@
+#### Integrations
+
+##### VMRay
+
+- Updated the Docker image to: *demisto/python3:3.10.12.68300*.
+
+- Added the ***vmray-get-screenshots*** command to download the screenshots for a specific analysis.
+
+#### Playbooks
+
+##### Detonate File - VMRay
+
+Fixed an issue in which the playbook input parameters *archive_password*, *document_password*, *max_jobs*, *sample_type*, *shareable*, and *tags* were not used by the ***vmray-upload-sample*** command.
+
+<!--
```

Comment:
If all the changes in Playbook are just `format`, it seems better to revert it and cancel the changes